app/scripts/modules/docker/src/pipeline/trigger/dockerTriggerOptions.directive.js

```diff
@@ -65,8 +65,22 @@ module.exports = angular
         }));
     };
 
-    this.updateSelectedTag = (item) => {
-      this.command.extraFields.tag = item;
+    this.updateSelectedTag = (tag) => {
+      this.command.extraFields.tag = tag;
+
+      if (this.command.trigger && this.command.trigger.repository) {
+        let imageName = '';
+        if (this.command.trigger.registry) {
+          imageName += this.command.trigger.registry + '/';
+        }
+        imageName += this.command.trigger.repository;
+        this.command.extraFields.artifacts = [{
+          type: "docker/image",
```

Comment:
This should be in single quotes, i.e. `type: 'docker/image'`

Reply:
Fixed